packages/playwright-core/src/utils/hostPlatform.ts

```diff
@@ -67,7 +66,7 @@ function calculatePlatform(): { hostPlatform: HostPlatform, isOfficiallySupporte
     if (distroInfo?.id === 'ubuntu' || distroInfo?.id === 'pop' || distroInfo?.id === 'neon' || distroInfo?.id === 'tuxedo') {
       const isOfficiallySupportedPlatform = distroInfo?.id === 'ubuntu';
       if (parseInt(distroInfo.version, 10) <= 19)
-        return { hostPlatform: ('ubuntu18.04' + archSuffix) as HostPlatform, isOfficiallySupportedPlatform };
+        return { hostPlatform: ('ubuntu18.04' + archSuffix) as HostPlatform, isOfficiallySupportedPlatform: false };
```

Comment:
It seems strange to cast `ubuntu18.04` to `HostPlatform` that does not list ubuntu 18 at all.